src/main/java/teammates/storage/api/FeedbackResponseCommentsDb.java

```diff
@@ -52,28 +52,11 @@ public FeedbackResponseCommentAttributes createFeedbackResponseComment(FeedbackR
                 "Trying to get non-existent FeedbackResponseComment, possibly entity not persistent yet.");
     }
 
-    /*
-     * Removes search document for the given comment
-     */
-    public void deleteDocument(FeedbackResponseCommentAttributes commentToDelete) {
-        Long id = commentToDelete.getId();
-
-        if (id == null) {
-            Key<FeedbackResponseComment> key = getEntityQueryKeys(commentToDelete).first().now();
-
-            if (key == null) {
-                return;
-            }
-
-            id = key.getId();
-        }
-
-        deleteDocument(Const.SearchIndex.FEEDBACK_RESPONSE_COMMENT, id.toString());
-    }
-
     /**
      * Removes search document for the comment with given id.
      *
+     * <p>See {@link FeedbackResponseCommentSearchDocument#toDocument()} for more details.</p>
```

Comment:
This kind of Javadoc should be `@see ...` instead.

Reply:
I feel that `@see` is more like providing links for similar methods. I put link `toDocument()` method here to give reference to how document id is generated.